src/mappings/Rewards.ts

```diff
@@ -92,6 +92,10 @@ async function handleRewardForTxHistory(rewardEvent: SubstrateEvent): Promise<vo
             eventRecord.event.method == rewardEvent.event.method) {
 
             let {event: {data: [account, _]}} = eventRecord
```

Comment:
What is the problem here? Looks like we need to check types of the events not the parameters. Then handle events differently

Reply:
Reward event difference. Same name and lookup, different attributes 

https://kusama.subscan.io/runtime/Staking?version=1050
https://kusama.subscan.io/runtime/Staking?version=1045